blog/forge-v6-release.md

```diff
@@ -0,0 +1,205 @@
+
+  - This change affects the init and import commands. It changes these commands to create a forge.config.js rather than creating config.forge in your package.json.
+  - The internal signature of `Plugin.getHook(name)` has been changed to `Plugin.getHooks(name)` [#2995](https://github.com/electron/forge/pull/2995) as part of these changes.
+  - This is considered a breaking change for any third-party templates because the base template no longer sets up a config.forge object, and any template-specific mutations to the Forge config should instead be performed on the forge.config.js file.
+  - templates that mutated the Forge config within `package.json` will need to instantiate their own `forge.config.js` or `forge.config.ts`
+
+### Upgraded Maker Wix dependency to `electron-wix-msi@5.0.0` ([3008](https://github.com/electron/forge/pull/3008)))
+  - This upgrade includes a rename from `appIconPath` to `icon` in the config ([#153](https://github.com/electron-userland/electron-wix-msi/pull/153)).
+
+### Upgraded required Node.js to 14 LTS ([#2921](https://github.com/electron/forge/pull/2921))
+
+### Upgraded package dependency to `electron-packager@17` ([#2978](https://github.com/electron-userland/electron-forge/pull/2978))
+
+The upgrade to Electron Packager 17 introduces the shiny new `@electron/osx-sign` package for macOS code signing. It's a rewrite of the old `electron-osx-sign` tool with more sensible defaults.
+
+To migrate, we recommend seeing if the default `packagerConfig.osxSign` options work for you and tweaking the default entitlements to your needs. Otherwise, see the `@electron/osx-sign` [MIGRATION.md](https://github.com/electron/osx-sign/blob/main/MIGRATION.md) doc for a 1:1 conversion from the old config options to the new ones.
+
+### Renamed Electron Rebuild config ([#2963](https://github.com/electron-userland/electron-forge/pull/2963))
+
+For consistency with the `packagerConfig` option for `electron-packager`, the field to configure `@electron/rebuild` has now been shortened to `rebuildConfig`.
+
+```diff
+{
+-  electronRebuildConfig: { /* ... */ }
++  rebuildConfig: { /* ... */ }
+}
+```
+
+### Renamed `identity-validation` to`identityValidation` ([#2959](https://github.com/electron-userland/electron-forge/pull/2959))
+
+The field to configure `identity-validation` has now been shortened to `identityValidation`.
+1
+```diff
+{
+-  identity-validation: ""
++  identityValidation: ""
+}
+```
+
+### Removed `@electron-forge/template-typescript` template ([#2948](https://github.com/electron-userland/electron-forge/commit/fc9421d513300b98c987af41ae71cb5d7e696fd1))
+
+- this has been removed in favor of the [Webpack + TypeScript Template].
+
+### Removed `lint` command ([#2964](https://github.com/electron/forge/pull/2964))
+
+### Removed `install` command ([#2958](https://github.com/electron-userland/electron-forge/commit/6b215b0c1d91c998bb2ab953b502e87868527ed9))
+
+## A call to action:
```

Comment:
```suggestion
## Feedback wanted!
```

Reply:
I think we should stick with `a call to action`, or modify this to `Submit your Feedback to us!` 

- employing a push-style call to action phrase is better to incite desired action from readers

Reply:
I wasn't sure if "A call to action" was a placeholder for "add a CTA here" LOL

Reply:
LOL v understandable- I'm gonna change this to `Submit your feedback!` so readers don't think the same way